jalib/jalloc.cpp

```diff
@@ -269,14 +271,14 @@ class JFixedAllocStack
     };
     struct StackHead {
       uintptr_t counter;
-      FreeItem* volatile node;
+      FreeItem* ATOMIC_SHARED node;
```

Comment:
It's existing code with this reversed order, but as part of this PR, can you please change the order to `ATOMIC_SHARED <type> var;`? Here and below.

Reply:
I tested, and moving ATOMIC_SHARED changes the meaning.  Right now, it's a pointer to a volatile 'node'.  If we reverse it, it becomes a volatile pointer to 'node'.  And of course, with that different meaning, it's inconsistent with how it's used.